iati_datastore/iatilib/frontend/api1.py

```diff
@@ -67,6 +67,32 @@ def about_dataset(dataset):
             resources=resources,
     )
 
+
+@api.route('/about/datasets/nest')
```

Comment:
Looking at this again, I'm wondering about the name... Would it be clearer if the API call was `about/datasets/fetch_status` ??

Reply:
done